test/spec/GitSpec.hs

```diff
@@ -0,0 +1,92 @@
+module GitSpec (spec) where
+import           Test.Hspec
+
+import           Control.Biegunka
+import           Control.Biegunka.Source.Git (git_)
+import           Control.Lens
+import           Data.Maybe                  (listToMaybe)
+import           System.Directory            (doesDirectoryExist)
+import           System.FilePath             ((</>))
+import           System.IO.Temp              (withSystemTempDirectory)
+import qualified System.Process              as P
+
+spec :: Spec
+spec = do
+  describe "git_" $ do
+
+    it ("creates new directory and set branch correctly " ++
+        "when local path doesn't exist") $
+      withBiegunkaDirectory $ \tmp -> do
```

Comment:
As `withBiegunkaDirectory` is going to be used in every example, it would be nice to move it into an `around` block.